presto-main/src/main/java/com/facebook/presto/server/QuerySessionSupplier.java

```diff
@@ -115,6 +119,11 @@ else if (context.getTimeZoneId() != null) {
             sessionBuilder.addSessionFunction(entry.getKey(), entry.getValue());
         }
 
+        // Put after setSystemProperty are called
+        WarningCollector warningCollector = warningCollectorFactory.create(
+                sessionBuilder.getSystemProperty(WARNING_HANDLING, WarningHandlingLevel.class));
```

Comment:
We can put these two into the same line. Our line cap is at around 180 chars

Reply:
Done